packages/rich-text/src/component/index.native.js

```diff
@@ -156,6 +165,16 @@ export class RichText extends Component {
 		return { ...value, start, end };
 	}
 
+	getHtmlForRecordCreation( value, tag ) {
+		if (
+			tag === 'p' &&
+			( ! value || value.length < 3 || ! value.startsWith( '<p>' ) )
+		) {
+			return '<p>' + value + '</p>';
+		}
+		return value;
+	}
+
```

Comment:
Removing empty paragraph tags to fix missing placeholders on Quote/Pullquote blocks seems to work fine except for calls to the imported create function from gutenberg/packages/rich-text/src/create.js. That function seems to only work at extracting text in order to create the record object if we re-add paragraph tags for multiline text.